test/knexfile.js

```diff
@@ -6,7 +6,7 @@ var _          = require('lodash');
 var Promise    = require('bluebird');
 
 // excluding oracle and mssql dialects from default integrations test
-var testIntegrationDialects = (process.env.DB || "maria mysql mysql2 postgres sqlite3").match(/\w+/g);
+var testIntegrationDialects = (process.env.DB || "postgres" || "maria mysql mysql2 postgres sqlite3").match(/\w+/g);
```

Comment:
Why is this here? locally you can use DB env variable to select which bds are tested. IIRC they are set in .travis configuration for knex CI.